### ISSUE TITLE
Some characters in the Name field like & are shown as html like &#38

### ISSUE THREAD

**Issue by dxdemetriou** [pluginsGLPI/formcreator#2988]
**Describe the bug**

**To Reproduce**
1. Go to 'Administration - Forms'
2. Select some Form
3. Go to 'Questions'
4. Click on some section's titlename
5. Write something containing the & character

**Expected behavior**
The character should be shown as ````&```` instead of ````&#38````

**Screenshots**
![image](https://user-images.githubusercontent.com/41939421/190924830-1090fbe6-df4e-4039-8dc7-8c79712ee1fa.png)
![image](https://user-images.githubusercontent.com/41939421/190924844-c60e3101-aa8d-4708-9849-4f3338835b2f.png)

**GLPI / Plugins (please complete the following information):**
- GLPI version: 10.0.3
- list of all plugins with their version (available in GLPI > Setup > General > tab System > bottom of the page)
````
	fields               Name: Additional fields              Version: 1.17.3     State: Installed / not activated               
		Install Method: Marketplace
	advancedplanning     Name: advancedplanning               Version: 1.1.0      State: Installed / not activated               
		Install Method: Marketplace
	datainjection        Name: Data injection                 Version: 2.12.0     State: Installed / not activated               
		Install Method: Marketplace
	escalade             Name: Escalation                     Version: 2.8.0      State: Installed / not activated               
		Install Method: Marketplace
	formcreator          Name: Form Creator                   Version: 2.13.0     State: Enabled                                 
		Install Method: Marketplace
	gappessentials       Name: Gapp Essentials                Version: 2.0.1      State: Installed / not activated               
		Install Method: Marketplace
	glpiinventory        Name: GLPI Inventory                 Version: 1.0.4      State: Enabled                                 
		Install Method: Marketplace
	itilcategorygroups   Name: ItilCategory Groups            Version: 2.5.1      State: Installed / not activated               
		Install Method: Marketplace
	mreporting           Name: More Reporting                 Version: 1.8.1      State: Installed / not activated               
		Install Method: Marketplace
	screenshot           Name: Screenshot                     Version: 2.0.1      State: Installed / not activated               
		Install Method: Marketplace
	stab                 Name: Split Timeline Action Buttons  Version: 1.0.2      State: Enabled                                 
		Install Method: Marketplace
	tag                  Name: Tag Management                 Version: 2.10.0     State: Enabled                                 
		Install Method: Marketplace
	typology             Name: Typologies                     Version: 3.0.0      State: Enabled                                 
		Install Method: Marketplace
	yagp                 Name: yagp                           Version: 2.0.0      State: Enabled                                 
		Install Method: Marketplace

````

**Desktop (please complete the following information):**
 - OS: Windows/Linux
 - Browser: Chrome
 - Version: 105

**Comment by btry** [comment]
Hi

Fixed with 491dcb69

**Comment by dxdemetriou** [comment]
> Hi
> 
> Fixed with [491dcb6](https://github.com/pluginsGLPI/formcreator/commit/491dcb69c0026117ca4e62363b0d9156be7e5cda)

Fixed on questions but the main title is affected also

1. Go to 'Administration - Forms'
2. Select some Form
3. Go to 'Form'
4. Write something containing the & character in the 'Name'

**Comment by btry** [comment]
I'l do a fix. Be aware it will not be included in the release 2.13.1 because it has been packaged for an imminent release.

**Comment by dxdemetriou** [comment]
> I'l do a fix. Be aware it will not be included in the release 2.13.1 because it has been packaged for an imminent release.

Yes, sure. I'll remember to make the changes manually until then. Thanks :)

**Comment by btry** [comment]
Here is a commit for https://github.com/pluginsGLPI/formcreator/issues/2988#issuecomment-1250747529 5b802658
